util/kafka/test-client.py

```diff
@@ -0,0 +1,114 @@
+from confluent_kafka import Producer, Consumer, KafkaError, KafkaException, Message
```

Comment:
Should we add the `confluent-kafka` Python dep somewhere? Is it just relevant for `util/kafka`?